framework/test/unit/modules/dpos_v2/module.spec.ts

```diff
@@ -425,4 +425,218 @@ describe('DPoS module', () => {
 			});
 		});
 	});
+
+	describe('_updateValidators', () => {
+		let dpos: DPoSModule;
+
+		beforeEach(async () => {
+			dpos = new DPoSModule();
+			await dpos.init({
+				generatorConfig: {},
+				genesisConfig: {} as GenesisConfig,
+				moduleConfig: defaultConfigs,
+			});
+		});
+
+		describe('given valid scenarios', () => {
+			const scenarios = [
+				forgerSelectionZeroStandbyScenario,
+				forgerSelectionOneStandbyScenario,
+				forgerSelectionTwoStandbyScenario,
+				forgerSelectionLessTHan103Scenario,
+				forgerSelectionMoreThan2StandByScenario,
+			];
+			const defaultRound = 5;
+
+			for (const scenario of scenarios) {
```

Comment:
Can't we use `describe.each` here?

Reply:
I think we can, but  i would keep it like  this for now. I just copied from the existing test =)